db/migrate/20221017163812_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] Same here, Please edit this reference because it's supposed to be `author_id` not `user_id` as a foreign key.